CourseGrab/Keys.swift

```diff
@@ -8,25 +8,14 @@
 
 import Foundation
 
-/**
- `Keys` is a list of expected hidden key value pairs in `Keys.plist`, used for sensitive
- developer information like API keys and secrets.
- */
-enum Keys: String {
-    case googleClientID = "google-client-id"
-
-    /**
-     `value` is the string representation of the key. Implicitly unwrapped because
-     the app should crash without Keys stored properly, while allowing you to check
-     for nil with `value == nil`.
-     */
-    var value: String! {
-        return Keys.keyDict[rawValue] as? String
-    }
+struct Keys {
+    
+    static let googleClientID = Keys.keyDict["google-client-id"] as? String
```

Comment:
We usually stay away from force unwraps, but in this context, do you all think it would be appropriate? Otherwise we'd have to unwrap it every single time we access it even though we know it won't be nil.

Reply:
It will be nil if someone who does not have the keys.plist clones the project. But even then the app would still crash on launch because there is no Google Client ID provided in AppDelegate. Ill force unwrap it for now, but this is something we should address in the future

Reply:
Agreed